### DISCUSSION TITLE
Readme Preview not working (shows blank panel) in Codespaces/VSCode

### DISCUSSION THREAD

**Discussion by valhuber** [community/community#32847] [Codespaces]
When I start [this project](https://github.com/ApiLogicServer/ApiLogicProject) in Codespaces, then (in VSCode) open the readme -> show preview, it shows as blank:

<img width="1307" alt="readme-preview" src="https://user-images.githubusercontent.com/5579631/189797756-6c2fa20c-ba14-4bf7-840d-60a0246018b7.png">

**Comment by dmilburn** [comment]
Hi @valhuber, thanks for reporting! I just gave your repo a try and was able to view the readme in a preview.  

<img width="1057" alt="Screen Shot 2022-09-13 at 11 14 08 AM" src="https://user-images.githubusercontent.com/10053402/189939722-0a11ffad-3aa8-4338-a98a-ff8d5e6f35d5.png">

I'm seeing you have an error in the bottom right of the screen so seems like maybe something was going wrong with your particular codespace. I will dig in and circle back. Could you also let me know what browser you're using please?

**Reply by valhuber** [reply]
@dmilburn How odd... I was using Safari.  I believe I also tried FireFox, with same result.

**Reply by valhuber** [reply]
@dmilburn Hmm, tried it again in Safari, worked this time (tho still seeing that funky terminal window reported earlier):

<img width="1436" alt="readme" src="https://user-images.githubusercontent.com/5579631/189941400-143826b2-c546-416f-b004-12defc4482c0.png">
  
Looks like we should close this unless/until it recurs.

**Reply by dmilburn** [reply]
how strange, thanks for reporting! i've added an issue for internal investigation but won't prioritize it as high as it seems to not be a consistent problem.

**Comment by valhuber** [comment]
Seems to be working now - see above

**Reply by GideonCheruiyot** [reply]
We will investigate further if the issue resurfaces. Thanks for reporting @valhuber
